.github/workflows/imagescan.yaml

```diff
@@ -0,0 +1,26 @@
+name: Security checks
```

Comment:
There is also gosec scanning going on elsewhere, so maybe we can use a name that's more specifically about container image scanning.